test/unit/connection/result/result_test_duplicated_columns.js

```diff
@@ -0,0 +1,151 @@
+  ];
+  const responseWithColumns = (columnRowSet) => {
+    return {
+      'data': {
+        'parameters': [{'name': 'CLIENT_PREFETCH_THREADS', 'value': 4}, {
+          'name': 'TIMESTAMP_OUTPUT_FORMAT',
+          'value': 'YYYY-MM-DD HH24:MI:SS.FF3 TZHTZM'
+        }, {'name': 'JS_TREAT_INTEGER_AS_BIGINT', 'value': false}, {
+          'name': 'TIME_OUTPUT_FORMAT',
+          'value': 'HH24:MI:SS'
+        }, {'name': 'CLIENT_RESULT_CHUNK_SIZE', 'value': 160}, {
+          'name': 'TIMESTAMP_TZ_OUTPUT_FORMAT',
+          'value': ''
+        }, {'name': 'CLIENT_SESSION_KEEP_ALIVE', 'value': false}, {
+          'name': 'CLIENT_OUT_OF_BAND_TELEMETRY_ENABLED',
+          'value': false
+        }, {'name': 'QUERY_CONTEXT_CACHE_SIZE', 'value': 5}, {
+          'name': 'CLIENT_METADATA_USE_SESSION_DATABASE',
+          'value': false
+        }, {
+          'name': 'ENABLE_STAGE_S3_PRIVATELINK_FOR_US_EAST_1',
+          'value': false
+        }, {'name': 'CLIENT_RESULT_PREFETCH_THREADS', 'value': 1}, {
+          'name': 'TIMESTAMP_NTZ_OUTPUT_FORMAT',
+          'value': 'YYYY-MM-DD HH24:MI:SS.FF3'
+        }, {
+          'name': 'CLIENT_METADATA_REQUEST_USE_CONNECTION_CTX',
+          'value': false
+        }, {'name': 'CLIENT_HONOR_CLIENT_TZ_FOR_TIMESTAMP_NTZ', 'value': true}, {
+          'name': 'CLIENT_MEMORY_LIMIT',
+          'value': 1536
+        }, {'name': 'CLIENT_TIMESTAMP_TYPE_MAPPING', 'value': 'TIMESTAMP_LTZ'}, {
+          'name': 'TIMEZONE',
+          'value': 'America/Los_Angeles'
+        }, {'name': 'CLIENT_RESULT_PREFETCH_SLOTS', 'value': 2}, {
+          'name': 'CLIENT_TELEMETRY_ENABLED',
+          'value': true
+        }, {'name': 'CLIENT_DISABLE_INCIDENTS', 'value': true}, {
+          'name': 'CLIENT_USE_V1_QUERY_API',
+          'value': true
+        }, {'name': 'CLIENT_RESULT_COLUMN_CASE_INSENSITIVE', 'value': false}, {
+          'name': 'CSV_TIMESTAMP_FORMAT',
+          'value': ''
+        }, {'name': 'BINARY_OUTPUT_FORMAT', 'value': 'HEX'}, {
+          'name': 'CLIENT_ENABLE_LOG_INFO_STATEMENT_PARAMETERS',
+          'value': false
+        }, {
+          'name': 'CLIENT_TELEMETRY_SESSIONLESS_ENABLED',
+          'value': true
+        }, {
+          'name': 'JS_DRIVER_DISABLE_OCSP_FOR_NON_SF_ENDPOINTS',
+          'value': false
+        }, {'name': 'CLIENT_FORCE_PROTECT_ID_TOKEN', 'value': true}, {
+          'name': 'CLIENT_CONSENT_CACHE_ID_TOKEN',
+          'value': false
+        }, {'name': 'DATE_OUTPUT_FORMAT', 'value': 'YYYY-MM-DD'}, {
+          'name': 'CLIENT_STAGE_ARRAY_BINDING_THRESHOLD',
+          'value': 65280
+        }, {'name': 'CLIENT_SESSION_KEEP_ALIVE_HEARTBEAT_FREQUENCY', 'value': 3600}, {
+          'name': 'AUTOCOMMIT',
+          'value': true
+        }, {'name': 'CLIENT_SESSION_CLONE', 'value': false}, {'name': 'TIMESTAMP_LTZ_OUTPUT_FORMAT', 'value': ''}],
+        'rowtype': columnRowSet,
+        'rowset': [['a', '1', 'a1', '2', 'a2', '3', 'a3', '4', 'a4', '5']],
```

Comment:
we are only interested in `rowtype` and `rowset` - do we need all other things to be stubbed?

Reply:
It could be empty array.